e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -169,9 +175,9 @@ private void loadDockerComposeContainers(int apiPort) {
                 .withScaledService("worker", 2)
                 .withScaledService("api", 1)
                 .withExposedService("api", DEFAULT_API_PORT, new HostPortWaitStrategy()
-                    .withStartupTimeout(Duration.of(200, SECONDS)));
-//                .withLogConsumer("worker", new Slf4jLogConsumer(log)) // Use to debug, for now there's too much log data
-//                .withLogConsumer("api", new Slf4jLogConsumer(log));
+                    .withStartupTimeout(Duration.of(200, SECONDS)))
+                .withLogConsumer("worker", new Slf4jLogConsumer(workerLogger)) // Use to debug, for now there's too much log data
+                .withLogConsumer("api", new Slf4jLogConsumer(apiLogger));
```

Comment:
Might want to recomment these out.

Reply:
Quick chat, Will wants to make these semi-permanent.  I'm fine with that.

Comment needs to be made more accurate or deleted.